setup.py

```diff
@@ -423,7 +423,7 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness, cmdOptions=None)
       ["extensions/cpp_region/PyRegion.cpp",
```

Comment:
Also, the travis job should have stopped due to failure at https://travis-ci.org/numenta/nupic/jobs/68411571#L1106, but it continued and failed for a different reason.